src/RunHub.jsx

```diff
@@ -11,8 +11,8 @@ export default function RunHub() {
 
         <div style={primaryActionsGridStyle}>
           <ActionButton label="Play Hand" onClick={() => navigate('/blackjack')} />
+          <ActionButton label="Select Deck" onClick={() => navigate('/deck-select')} />
```

Comment:
I don't think the player should be able to select a deck once they start a run. This is like the part of the roguelike where you are customizing your character. The choices you make would be meaningless if you could change it at any time.